README.md

```diff
@@ -1,123 +1,3 @@
-# hello-microservice
+# doppler-account-plans
```

Comment:
It is out of the scope of this PR (_"Add account plans controller"_).

I can live with that, but do not forget to search other places where `hello` have not been replaced, for example, `.vscode/tasks.json`

Reply:
Done! resolving the conversation.